docs/security-council-members.md

```diff
@@ -0,0 +1,40 @@
+---
+id: security-council-members
+title: 'Security Council Members'
+sidebar_label: Security Council Members
+description: Current members of the security council
+dao_author: dzgoldman
+dao_sme: dzgoldman
+---
+
+The following are the current members of the Security Council; for information about the Security Council, see ["Security Council Overview"](./concepts/security-council).
+
+i. September Cohort
+
+1.  [0xhombre](https://twitter.com/0xhombre)
+    - `0xb71ca4ffbb7b58d75ba29891ab45e9dc12b444ed`
+2.  [Matt Fiebach](https://twitter.com/MattFiebach)
+    - `0x3bd8e2ac65ad6f0f094ba6766cbd9484ab49ef23`
+3.  [Harry Kalodner](https://www.linkedin.com/in/hkalodner/) is Co-Founder and CTO at Offchain Labs. Harry started working on building Arbitrum while studying at Princeton University.
+    - `0xf8e1492255d9428c2Fc20A98A1DeB1215C8ffEfd`
+4.  [Patrick McCorry](https://twitter.com/stonecoldpat0) has spent most of his adult life working on cryptocurrencies and his time was split across academic research and industry. Before working at the Arbitrum Foundation, he has worked on transaction delivery, layer-2 protocols, atomic swaps, consensus protocols, and applied cryptography. In a past life, he was an Assistant Professor at King’s College London and an accomplished researcher at UCL, UIUC and NCL.
+    - `0x8f10e3413586c4a8dcfce19d009872b19e9cd8e3`
+5.  [John Morrow](https://twitter.com/jmo_mx)
+    - `0x3e286452b1c66abb08eb5494c3894f40ab5a59af`
+6.  [Omer](https://twitter.com/omeragoldberg)
+    - `0xb07dc9103328a51128bc6cc1049d1137035f5e28`
+
+ii. March Cohort
+
+1.  [Rachel Bousfield](https://github.com/rachel-bousfield) has been a software engineer at Offchain Labs since 2021. She is currently leading the development of Stylus.
+    - `0x5A1FD562271aAC2Dadb51BAAb7760b949D9D81dF`
+2.  [Justin Drake](https://www.linkedin.com/in/drakefjustin) has been a researcher at the Ethereum Foundation since 2017. He focuses on Ethereum consensus layer upgrades.
+    - `0x5280406912EB8Ec677Df66C326BE48f938DC2e44`
+3.  [Bartek Kiepuszewski](https://www.linkedin.com/in/bartek-kiepuszewski-b3a299/?originalSubdomain=pl) has been a blockchain architect at MakerDAO since 2017. He also co-founded [l2beat.com](http://l2beat.com/) and TokenFlow Insights. Bartek holds a PhD in computer science from Queensland University of Technology.
```

Comment:
```suggestion
3.  [Bartek Kiepuszewski](https://twitter.com/bkiepuszewski) has been a blockchain architect at MakerDAO since 2017. He also co-founded [l2beat.com](http://l2beat.com/) and TokenFlow Insights. Bartek holds a PhD in computer science from Queensland University of Technology.
```